build/yamls/base/controller-rbac.yml

```diff
@@ -87,10 +87,20 @@ rules:
       - apiservices
     resourceNames:
       - v1beta1.system.antrea.tanzu.vmware.com
-      - v1beta1.networking.antrea.tanzu.vmware.com
+      - v1beta1.controlplane.antrea.tanzu.vmware.com
     verbs:
       - get
       - update
+  - apiGroups:
+      - apiregistration.k8s.io
+    resources:
+      - apiservices
+    resourceNames:
+      # Add the APIServices for the deprecated APIGroups here. antrea-controller
```

Comment:
Thanks for adding the comment!

Reply:
do we have a timeline in terms of releases for how long we plan to attempt deleting old apiservices? ie after X releases we will no longer clean up..